tests/publiccloud/prepare_tools.pm

```diff
@@ -71,14 +71,23 @@ sub run {
     ensure_ca_certificates_suse_installed();
 
     # Install prerequisite packages test
-    zypper_call('-q in python310-pip python310-devel python3-pytest python3-img-proof python3-img-proof-tests podman docker jq rsync');
+    zypper_call('-q in python310-pip python310-devel python3-pytest python3-img-proof python3-img-proof-tests podman docker jq rsync unzip');
     record_info('python', script_output('python --version'));
     systemctl('enable --now docker');
     assert_script_run('podman ps');
     assert_script_run('docker ps');
 
     # Install AWS cli
-    install_in_venv('aws', requirements => 1);
+    my $aws_version = '2.12.3';
+    # Download and import the AWS public PGP key
+    assert_script_run(sprintf('curl -f -v %s/data/publiccloud/aws.asc -o /tmp/aws.asc', autoinst_url()));
```

Comment:
why we importing this key from our repo ? usually such things are shipped together with package ? from where you got this key ?

Reply:
From their [official documentation](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
![image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/1254493/56ceea61-c054-47c8-b003-9dbe3c0c1b9f)

I decided to download it as it's more 'long term' compared to the package itself and also it plays role in the verification process of the package so I thought that it's better to not download everything.